pkg/crd/testdata/golden/with-header-file/bar.example.com_foos.yaml

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
It is an yaml file. So, would not the comments need to be with `#` as prefix?

Reply:
lol - good point!  I'll fix :-)

Reply:
Fixed - thanks for spotting this!